devops_sccs/core.py

```diff
@@ -77,19 +77,24 @@ async def create(cls, config={}):
         self.enableHook = config.get("hook_server") is not None
         if self.enableHook :
             self.hookServer = HookServer(config["hook_server"])
-            
-        await self.load_builtin_plugins(config.get("plugins", {}))
+            #we need to lock it up here , otherwise the endpoints won't register.
+            await self.hookServer.lock.acquire()   
+        val = await self.load_builtin_plugins(config.get("plugins", {}))
 
-        await self.load_external_plugins(config.get("plugins", {}))
+        val = await self.load_external_plugins(config.get("plugins", {}))
         
-        if self.enableHook :
-            self.hookServer.start_server()
+        if self.enableHook & val :

```

Comment:
```suggestion
        if self.enableHook and val :

```
p-e que ca fonctionne aussi avec le bitwise `&`, mais je pense que c'est plus clair avec `and`